packages/loot-core/src/server/budget/goaltemplates.ts

```diff
@@ -406,7 +418,9 @@ async function applyCategoryTemplate(
       case 'spend': {
         // spend has 'amount' and 'from' and 'month' params
         let from_month = new Date(`${template.from}-01`);
+        from_month = addMinutes(from_month, from_month.getTimezoneOffset());
```

Comment:
Do you think it would be reasonable to write a function that does both of these in one step (since it is common to do them both in this file)?

Reply:
I think that's reasonable.  I'll make that change as well.